### PR TITLE
chapter 07 Graphics

### PR DESCRIPTION
**Pull request by fgazzelloni**

### CODE REVIEW

File: 07-graphics.Rmd [hunk 1]
```diff
@@ -1,4 +1,305 @@
-# Graphics and Shiny
+# Graphics
+
+**Learning objectives**
+
+- how to use `renderPlot()`
```
Comment (jonthegeek) [inline 07-graphics.Rmd]:
```suggestion
- Use `renderPlot()` to display reactive plots.
```

File: 07-graphics.Rmd [hunk 2]
```diff
@@ -1,4 +1,305 @@
...
+- create interactive plots
```
Comment (jonthegeek) [inline 07-graphics.Rmd]:
```suggestion
- Create interactive plots.
```

File: 07-graphics.Rmd [hunk 3]
```diff
@@ -1,4 +1,305 @@
...
+- displaying images with `renderImage()`
```
Comment (jonthegeek) [inline 07-graphics.Rmd]:
```suggestion
- Display images with `renderImage()`.
```

**Review by jonthegeek — CHANGES_REQUESTED** [review]
A few tweaks to standardize the LOs. I'll commit these in a sec.

**Review by jonthegeek — APPROVED** [review]
Ok, that should do it!

**Comment by Fgazzelloni** [comment]
I opened a new branch and deleted the old one, but now how I push it again?

**Pull request merged**